_posts/2021-05-17-hhvm-4.110.markdown

```diff
@@ -0,0 +1,45 @@
+---
+title: "HHVM 4.110"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.110 is released! This release marks the end of support for 4.104;
+HHVM 4.105&ndash;4.109 remain supported, as do the 4.80 and 4.102 LTS releases.
+
+# Highlights
+
+- Fixed behavior of `setlocale()` when called
+  with the special values `""` or `"0"`
+  ([#8834](https://github.com/facebook/hhvm/pull/8834)).
```

Comment:
More details at https://hhvm.com/blog/2021/05/11/hhvm-4.109.html under future changes - might be worth mentioning that the default locale will now consistently be `"C"`, where previously it was... weird